opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -25,6 +25,11 @@
      */
     ScopeManager scopeManager();
 
+    /**
+     * @return the active {@link Span}, if any. This is a shorthand for {@link Tracer#scopeManager()#active()#span()}.
```

Comment:
Please also document the `null` check.
Otherwise people may either think this method could be unsafe to call in absence of active scope, or (worse) may think it's safe to unconditionally dereference active() scope in any situation.

Reply:
Good catch, will do.